tests/regression/tests/REQUEST-930-APPLICATION-ATTACK-LFI/930100.yaml

```diff
@@ -21,3 +21,18 @@
               uri: "/"
             output:
               log_contains: id "930100"
+    -
+      test_title: 930100-2
+      desc: "Path Traversal Attack Triple Dot (/.../)"
+      stages:
+        -
+          stage:
+            input:
+              dest_addr: "127.0.0.1"
+              method: "GET"
+              port: 80
+              headers:
+                  Host: "localhost"
+              uri: "/?foo=.../.../WINDOWS/win.ini"
```

Comment:
Should the test for this rule use encoded dots and forward slashes to match the rule's regular expression?

Reply:
I also added a test with encoded slashes and dots.